tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+        storage_ns = config.ENV_DATA["cluster_namespace"]
+
+        targets = {
+            "rook-ceph-mon-a": 3300,
+            "noobaa-mgmt": 443,
+        }
+
+        for svc_name, port in targets.items():
+            svc_ip = get_service_ip(svc_name, namespace=storage_ns)
+            if not svc_ip:
+                logger.warning(
+                    f"Service {svc_name} not found, skipping"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)